doc/newsfragments/fix_filename_linenumber.feature.rst

```diff
@@ -0,0 +1,3 @@
+* Upgraded plotly.js from 1.58.4 to 1.58.5
+* Added padding and fontFamily properties to css styling for timeInfo
+* Updated AssertionHeader.test.js.snap
```

Comment:
@kn-ms do you think this is a bit too detail from user perspective?

Reply:
Yes I do, I prefer something that describe the effect for user experience. Maybe:

 * Common styling for path and time of assertions